python-backend/app/__init__.py

```diff
@@ -80,15 +80,3 @@ def get(self):
     def default_error_handler(error):
         _, value, traceback = sys.exc_info()
         return json.loads({"error": str(traceback)})
-
-
-def register_apm(app):
```

Comment:
Moved this to extensions so that the init is a single line. Also so that people can now import APM module from extensions and do stuff like `apm.log(AAHHHHH)` for custom exceptions